src/containers/Guide/GuideAccounts/index.tsx

```diff
@@ -54,26 +53,25 @@ const GuideAccounts: FC = () => {
         <li>Made public to the network</li>
         <li>Acts as a receiving address</li>
         <ul>
-          <li>Other users can send coins to you using your account number</li>
+          <li>Any user can send coins to the owner of the account using this account number</li>
```

Comment:
That one I think you are right, maybe "their", although it would;t clarify if we refer to the sender or the recipient. "You", however, is not used elsewhere. I am good either way with that, doesn't matter much